rsmtool/notebooks/evaluation.ipynb

```diff
@@ -101,7 +114,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "Markdown(\"Confusion matrix using {} trimmed rounded scores (rows=system, columns=human).\".format(raw_or_scaled))"
+    "markdown_str = [\"Confusion matrix using {} trimmed rounded scores and human scores (rows=system, columns=human).\".format(raw_or_scaled)]\n",
+    "\n",
+    "if continuous_human_score:\n",
+    "    markdown_str.append(\"*Human scores have beeen rounded to the nearest integer.*\")\n",
```

Comment:
I would remove the italics here and start the sentence with "Note that ..." to be consistent with the other sections.